website/static/css/pages/login-page.css

```diff
@@ -38,3 +38,12 @@
     background-color: #CCC;
     border-color: #BBB;
 }
+.existing-user-signin {
+    /*height: 320px;*/
+    background-color: #EEE;
+    border-radius: 4px;
+    margin-top: 75px;
+}
+.existing-user-signin-button {
+    margin-right: 0px !important;
```

Comment:
This isn't a blocker, but see if you can refactor the markup to use Bootstarp's grid and get rid of this custom CSS.